var/spack/repos/builtin/packages/igprof/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Igprof(CMakePackage):
+    """IgProf (the Ignominous Profiler) is a simple nice tool for measuring and
+       analysing application memory and performance characteristics.
+
+       IgProf requires no changes to the application or the build process. It
+       currently works on Linux (ia32, x86_64)."""
+
+    homepage = "https://igprof.org/"
+    url      = "https://github.com/igprof/igprof/archive/v5.9.16.tar.gz"
+
```

Comment:
Do you want to add yourself as a Spack package maintainer for this package?  That would give you a chance to be notified of and comment on any changes to this package.

Reply:
I would highly doubt any changes will happen (last commit was ages ago :) ). Anyway, the answer is "not yet".